### ISSUE TITLE
Evaluating Llama-70b on ARC-e/c

### ISSUE THREAD

**Issue by obhalerao97** [facebookresearch/llama#933]
Hello, 
I'm trying to reproduce the results the paper mentions for ARC-e/c, but I'm getting a accuracy of 38.3 on ARC-c, where as the paper mentions an accuracy of 57.4.

I tried two methods since this is a MCQ dataset:
1) Extracting the output from the generated text
2) Calculating logits (same as what lm-eval-harness does)

The first method didn't work out too well, since the model would generate randomly formatted outputs and answer questions that were out of the choices given. The logits method gives me a 38.3% accuracy. Could you guide me to the correct method?

Much appreciated,
Thank you!

**Comment by obhalerao97** [comment]
I did use EleutherAI, but I'm getting an accuracy of 52.9 on the 70b llama 2 pretrained model, where as the paper reports 57.4, not sure how to explain the discrepancy between the scores.
